### PR TITLE
Tutorial based on jd feedback

### PR DESCRIPTION
**Pull request by alixdamman**

### CODE REVIEW

File: doc/source/tutorial/getting_started.rst [hunk 1]
```diff
@@ -33,86 +49,78 @@ You can create an array from scratch by supplying data, axes and optionally some
...
+    pop.meta.source = 'Eurostat'
 
-        pop.info
+    # display metadata
+    pop.meta
 
-# TODO: move this in a section of the tutorial (inspecting arrays)
+To get a short summary an array, type:
 
-* number of dimensions
```
Comment (gdementen) [inline doc/source/tutorial/getting_started.rst]: you remove this, from getting started but did not add it back in another section?

**Comment by alixdamman** [comment]
Let's try Review NB : https://app.reviewnb.com/larray-project/larray/pull/752/files/

File: doc/source/tutorial/tutorial_presenting_larray_objects.ipynb [hunk 2]
```diff
@@ -363,7 +363,7 @@
    "cell_type": "markdown",
```
Comment (gdementen) [inline doc/source/tutorial/tutorial_presenting_larray_objects.ipynb]: <p>elementsof ?</p>

 _Reply via <a href='https://app.reviewnb.com/larray-project/larray/pull/752/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Comment (gdementen) [reply doc/source/tutorial/tutorial_presenting_larray_objects.ipynb]: Hmmm, never mind... This seems to be a bug in ReviewNB :(

Comment (alixdamman) [reply doc/source/tutorial/tutorial_presenting_larray_objects.ipynb]: Sounds like it is. Unfortunately.

**Comment by alixdamman** [comment]
OK to push?

**Comment by gdementen** [comment]
ok

**Pull request merged**